OracleManagementAgent/dockerfiles/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+pushd ./latest
+docker build -t oracle/mgmtagent-container .
```

Comment:
Why isn't this script checking to see if the files you need to build are in place before starting the build? I'm also not sure why you need this and a `docker-compose.yml`